src/main/kotlin/com/chattriggers/ctjs/engine/langs/js/JSLoader.kt

```diff
@@ -319,7 +319,8 @@ object JSLoader : ILoader {
         JSContextFactory.optimize = true
     }
 
-    class CTRequire(moduleProvider: ModuleScriptProvider) : Require(moduleContext, scope, moduleProvider, null, null, false) {
+    class CTRequire(moduleProvider: ModuleScriptProvider) :
+        Require(moduleContext, scope, moduleProvider, null, null, false) {
```

Comment:
I'm not a fan of this style, since the super declaration isn't really separated from the class body. In this situation I typically write
```suggestion
    class CTRequire(
        moduleProvider: ModuleScriptProvider,
    ) : Require(moduleContext, scope, moduleProvider, null, null, false) {
```